src/datapump/sync/fire_alerts.py

```diff
@@ -30,90 +27,77 @@
 
 
 def process_active_fire_alerts(alert_type):
+    nrt_s3_directory = f"nasa_{alert_type.lower()}_fire_alerts/{VERSIONS[alert_type]}/vector/epsg-4326/tsv/near_real_time"
+    last_saved_date, last_saved_min = _get_last_saved_alert_time(nrt_s3_directory)
+    LOGGER.info(f"Last saved row datetime: {last_saved_date} {last_saved_min}")
+
     LOGGER.info(f"Retrieving fire alerts for {alert_type}")
-    response = requests.get(ACTIVE_FIRE_ALERTS_48HR_CSV_URLS[alert_type])
+    rows = []
+    with shapefile.Reader(ACTIVE_FIRE_ALERTS_7D_SHAPEFILE_URLS[alert_type]) as sf:
```

Comment:
Do you know if you will get an exception if you get a non-200 response from the http request to get the shapefile?  Just wanted to make sure we can distinguish between not being able to access the URL for some reason, vs there are no new rows that we haven't fetched.

Reply:
We do! If we get a non-200 response or the file is empty we get something like this:
ShapefileException: Unable to open https://firms.modaps.eosdis.nasa.gov/data/active_fire/suomi-npp-viirs-c2/shapes/zips/foo.dbf or https://firms.modaps.eosdis.nasa.gov/data/active_fire/suomi-npp-viirs-c2/shapes/zips/foo.shp.
If there are records but we've already seen them all I deal with that later:
raise Exception(
    f"{alert_type} shapefile contained no new records since {last_saved_date} {last_saved_min}"
)